homeassistant/components/tis_control/config_flow.py

```diff
@@ -0,0 +1,70 @@
+"""Config flow for TISControl integration."""
+
+from __future__ import annotations
+
+import logging
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_PORT
+from homeassistant.core import callback
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class TISConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for TISControl."""
+
+    VERSION = 1
+
+    async def async_step_user(self, user_input: dict | None = None) -> ConfigFlowResult:
+        """Handle a flow initiated by the user."""
+        errors: dict[str, str] = {}
+        if user_input is not None:
+            unique_id = f"tis_control:{user_input[CONF_PORT]}"
+            await self.async_set_unique_id(unique_id)
+            self._abort_if_unique_id_configured()
```

Comment:
I don't think you are testing for duplicate entries, you should also test that the config flow can successfully complete.